event-svc/src/event/validator/event.rs

```diff
@@ -0,0 +1,151 @@
+use std::sync::Arc;
+
+use ceramic_core::{Cid, EventId};
+use ceramic_event::unvalidated;
+use ipld_core::ipld::Ipld;
+use recon::ReconItem;
+
+use crate::{
+    event::{service::InvalidItem, DeliverableRequirement},
+    store::{EventInsertable, SqlitePool},
+    Result,
+};
+
+#[derive(Debug)]
+pub struct ValidatedEvents {
+    /// These events are valid
+    pub valid: Vec<ValidatedEvent>,
+    /// We don't have enough information to validate these events yet.
+    /// e.g. The init event is required and the node has not yet discovered it.
+    pub pending: Vec<UnvalidatedEvent>,
+    /// Events that failed validation
+    pub invalid: Vec<InvalidItem>,
```

Comment:
yea, fair. that was the name I used in the service a while ago because it's a response that represents something invalid and I named it to correspond to the `ReconItem` input.. which is also a dubious name

Reply:
went with these renames, lmk what you think

```rust
    pub valid: Vec<ValidatedEvent>,
    pub unvalidated: Vec<UnvalidatedEvent>,
    pub invalid: Vec<ValidationError>,